README.md

```diff
@@ -141,7 +141,7 @@ bin-target = "my-bin-name"
 
 # Enables additional file hashes on outputted css, js, and wasm files
 #
-# Optional: Defaults to false. Can also be set with the LEPTOS_HASH_FILES=false env var
+# Optional: Defaults to false. Can also be set with the LEPTOS_HASH_FILES=ON/OFF env var
```

Comment:
Because `LEPTOS_WATCH` uses `ON` I changed `LEPTOS_HASH_FILES` to `ON` / `OFF` to be consistent